### ISSUE TITLE
cant startup my app when i use log4js

### ISSUE THREAD

**Issue by 963542298** [log4js-node/log4js-node#1288]
node-version: 14.15.0

i  use vite start my app, then my app will throw error, i use webpack build my app, also throw error
![image](https://user-images.githubusercontent.com/45931289/178460565-2984598d-1afb-424f-9a41-a6432ef97527.png)
![image](https://user-images.githubusercontent.com/45931289/178460703-2a286673-9fdf-42ba-8314-8e0b4ab8d6b2.png)

**Comment by lamweili** [comment]
It appears that you are using in the front-end browser?

**Comment by 963542298** [comment]
yeah，i want to use this apply to my app！

**Comment by lamweili** [comment]
Can you provide the configuration that you passed into `log4js.configure()`?

**Comment by 963542298** [comment]
i dont configuration log4js, i try to use log4js in my web app, just run this code, then my app will throw these errors.
 ```javascript
import * as log4js from 'log4js';

log4js.getLogger().info("some log")
```

i will search ```browserify```  in issuse list. if cant solve this problem，I'm going to ask you
thank your

**Comment by lamweili** [comment]
Without configuration, below is the default:
https://github.com/log4js-node/log4js-node/blob/39218cc8d0e42773dd5513ca3d24cb397d0b4d50/lib/log4js.js#L144-L154

And if you are using it in a browser, the browser does not have a `stdout` appender, which uses Node.js runtime.
A browser does not have the Node.js library, `process`, to do `process.stdout.write()`.

You might need to configure with `console` appender to have a chance of working with `browserify`.

**Comment by 963542298** [comment]
today, i will try.

**Comment by lamweili** [comment]
@963542298 Did you fix it? Mind sharing for others?

**Comment by lamweili** [comment]
Thanks to @aellerton who came up with PR #1374 which resolved the issue.
Will be released in `log4js@6.8.1`.

**Comment by lamweili** [comment]
Thanks to @aellerton who came up with PR #1374 which resolved the issue.
Will be released in `log4js@6.9.0`.
